sdk/python/kfp/v2/compiler_cli_tests/test_data/pipeline_with_nested_conditions_yaml.json

```diff
@@ -270,6 +270,36 @@
         },
         "executorLabel": "exec-print"
       },
+      "comp-print-2": {
```

Comment:
Ohh...
Technically we could dedup these, but I think it's OK at this moment.
We need to add optional (and default) to MP and stop needing this workaround.

Reply:
Agree, and opened an internal feature request.